tapir/coop/models.py

```diff
@@ -722,6 +722,15 @@ def populate(
 
 
 class MembershipResignation(models.Model):
+    class ResignationType(models.TextChoices):
+        BUY_BACK = "coop_buys_shares_back", _(
+            "The cooperative buys the shares back from the member"
+        )
+        GIFT_TO_COOP = "gift_to_coop", _(
+            "The member gifts the shares to the cooperative"
+        )
+        TRANSFER = "transfer", _("The shares get transferred to another member")
+
```

Comment:
I think I got it now. You use models.TextChoices as enum. Right?

Reply:
Exactly, see https://docs.djangoproject.com/en/3.2/ref/models/fields/#enumeration-types